lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -510,8 +519,9 @@ def make_datatable(self):
 
         for course in self.get_courses():
             gdir = course.id.course
+            version = int(course.edited_on.strftime('%s'))
```

Comment:
@amir-qayyum-khan `strftime('%s')` will pull return the number of seconds since the Epoch, so it will be reliably unique. 

I think maybe you read that as `strftime('%S')` (capital S) which returns a number between 0 and 60.